google/cloud/spanner/commit_options.cc

```diff
@@ -30,13 +30,17 @@ CommitOptions::CommitOptions(Options const& opts)
   if (opts.has<TransactionTagOption>()) {
     transaction_tag_ = opts.get<TransactionTagOption>();
   }
+  if (opts.has<MaxCommitDelayOption>()) {
+    max_commit_delay_ = opts.get<MaxCommitDelayOption>();
+  }
```

Comment:
Please re-order the initialization to match the `CommitOptions` field declaration order.

Reply:
I moved the max_commit_delay_ field to the end, so this now matches, but a change here is no longer necessary.